### ISSUE TITLE
How to implement Allow/deny Alert for every Chat dialog while creating.

### ISSUE THREAD

**Issue by ajay2c9** [QuickBlox/quickblox-ios-sdk#658]
Hi,
      I am using quickblox in my project. Every time Allow/deny alert need to display for other chat users in the chat dialog when dialog is created.
When allow is selected chat need to continue. 
When Deny is selected, The user is added as blocked user and chatroom need to removed to that user.

How to implement the users blocked and unblocked feature.

Please give suggestions.
Thank you

**Comment by Pro100Andrey** [comment]
Hi @ajay2c9, is it public a room ?

**Comment by Pro100Andrey** [comment]
Hi @ajay2c9, 

>> When Deny is selected :
```objc
QBChatDialog *groupChatDialog = ...;
[groupChatDialog leaveWithCompletionBlock:^(NSError * _Nullable error) {
 
}];
```

**Comment by ajay2c9** [comment]
Hi Pro100Andrey,
Thank you for reply. How to display the alert for other users.

**Comment by Raikerian** [comment]
You mean notification in chat? Send a message in chat with custom parameter clarifying that this is notification message. And on receiver side just show it as notification.

**Comment by ajay2c9** [comment]
Ok, How to know that user accepted the request or not. Ex : After creating the dialog, by Notification message i displayed Alert with actions Allow or deny. If the user doesn't perform any action on alert. How to identify in this case.

**Comment by Raikerian** [comment]
Send message in chat when you are accepting contact request.

**Comment by Raikerian** [comment]
Closing as an issue never existed and all questions were answered.
If you are still having any other questions just create a new issue.
